app/src/main/java/com/sci/recipeandroid/feature/onboarding/ui/model/OnBoardModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.sci.recipeandroid.feature.onboarding.ui.model
+
+data class OnBoardModel(
+    var title : String ?= "",
```

Comment:
if property not mutable then plz use immutable val

Reply:
Update in e2ceaf1519d2ad526bf9d419752063a820c4ad13